.github/workflows/scheduled-build.yml

```diff
@@ -0,0 +1,73 @@
+# Daily build for testers at 5 AM UTC (8 AM Moscow Time, 11 AM Bishkek Time)
+on:
+  schedule:
+    - cron: '0 5 * * *'
+
+jobs:
+  android-build-artifacts:
+    name: "Run build feature artifacts"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout to the current commit hash"
+        uses: actions/checkout@v3
+        with:
+          ref: develop
+          fetch-depth: 0
+          token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+      - name: "Initializing firebase configs"
+        uses: ./.github/actions/setup-google-services
+        with:
+          google-services-content: ${{ secrets.GOOGLE_SERVICE_JSON_SECRET }}
+      - name: "Initialize project keys"
+        uses: ./.github/actions/setup-project
+        with:
+          amplitude-key-debug: ${{ secrets.AMPLITUDE_KEY_DEBUG }}
+          amplitude-key-release: ${{ secrets.AMPLITUDE_KEY_RELEASE }}
+          torus-verifier-debug: ${{ secrets.TORUS_VERIFIER_DEBUG }}
+          torus-verifier-feature: ${{ secrets.TORUS_VERIFIER_FEATURE }}
+          torus-verifier-release: ${{ secrets.TORUS_VERIFIER_RELEASE }}
+          rpc-pool-key: ${{ secrets.RPC_POOL_API_KEY }}
+          moonpay-key: ${{ secrets.MOONPAY_KEY }}
+          moonpay-sandbox-key: ${{ secrets.MOONPAY_SANDBOX_KEY }}
+          intercom-key: ${{ secrets.INTERCOM_API_KEY }}
+          intercom-app-id: ${{ secrets.INTERCOM_APP_ID }}
+          apps-flyer-key: ${{ secrets.APPS_FLYER_KEY }}
+          compare-key: ${{ secrets.COMPARE_PUBLIC_KEY }}
+          sentry-auth-token: ${{ secrets.SENTRY_AUTH_TOKEN }}
+          sentry-auth-dsn: ${{ secrets.SENTRY_AUTH_DSN }}
+          lokalise-key: ${{ secrets.LOKALISE_KEY }}
+          lokalise-app-id: ${{ secrets.LOKALISE_APP_ID }}
+      - name: "Generate changelog"
+        run: |
+          touch CHANGELOG.txt
+          echo "daily build \n branch - develop" >> CHANGELOG.txt"
+          echo "changelog.txt created"
+      - name: "Send start deploy message to slack"
+        run: bash bash_scripts/slack_deploy_start_message.sh ${{ secrets.SLACK_BOT_TOKEN }} ${{ secrets.SLACK_CHANNEL_ID }}
+      - name: "Generate a unique version build"
+        uses: ./.github/actions/setup-version-build
+      - name: "Build feature APK"
```

Comment:
could you add whitespaces between jobs?

Reply:
I just recently removed it. 
I thinks it's better to read when you minimized by blocks. 
But will see